front/components/poke/assistants/columns.tsx

```diff
@@ -142,3 +149,37 @@ async function archiveAssistant(
     window.alert("An error occurred while archiving the agent configuration.");
   }
 }
+
+async function restoreAssistant(
+  owner: WorkspaceType,
+  reload: () => void,
+  agentConfiguration: AgentConfigurationDisplayType
+) {
+  if (
+    !window.confirm(
+      `Are you sure you want to restore the ${agentConfiguration.name} assistant?`
+    )
+  ) {
+    return;
+  }
+
+  try {
+    const r = await fetch(
+      `/api/poke/workspaces/${owner.sId}/agent_configurations/${agentConfiguration.sId}/restore`,
+      {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+      }
+    );
+    if (!r.ok) {
+      throw new Error("Failed to restore agent configuration.");
+    }
+
+    reload();
```

Comment:
I'll tackle this in another PR but since you introduced the poke hooks for agent configuration we can remove all those `router.reload` calls.